dev-docs/bidders/vox.md

```diff
@@ -2,6 +2,9 @@
 layout: bidder
 title: VOX
 description: Prebid VOX Bidder Adapter
+schain_supported: true
+floors_supported: true
+userIds: all
 pbjs: true
 media_types: banner, video
```

Comment:
While you are at it: We recommend adding the following fields as well

- safeframes_ok: true/false
- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid

Reply:
@muuki88 
1. hard to say if we ok on safeframes) Some ad unit types like usual display banner ok with safeframes, while inImage is not supports safeframe
2. About the second param - multiformat_supported. Can you please show me where description of this param can be found?